src/go/rdctl/cmd/snapshotUnlock.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	p "github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/spf13/cobra"
+)
+
+var snapshotUnlockCmd = &cobra.Command{
+	Use:   "unlock",
+	Short: "Remove snapshot lock",
+	Long: `If an error occurs while doing a snapshot operation, sometimes the
+filesystem lock used to prevent simultaneous snapshot operations
+can be left behind. It then becomes impossible to work with
+snapshots. This command removes the filesystem lock. You should
+not have to use it under normal circumstances.`,
```

Comment:
```suggestion
not need to use it under normal circumstances.`,
```
I _think_ that's slightly less ambiguous, but I'm not sure…